specification/search/data-plane/Azure.Search/readme.md

```diff
@@ -545,10 +581,10 @@ require: $(this-folder)/../../../../profiles/readme.md
 
 # all the input files across all versions
 input-file:
-  - $(this-folder)/preview/2019-05-06-preview/searchservice.json
-  - $(this-folder)/preview/2019-05-06-preview/searchindex.json
-  - $(this-folder)/stable/2019-05-06/searchservice.json
-  - $(this-folder)/stable/2019-05-06/searchindex.json
+  - $(this-folder)/preview/2020-06-30-preview/searchservice.json
+  - $(this-folder)/preview/2020-06-30-preview/searchindex.json
+  - $(this-folder)/stable/2020-06-30/searchservice.json
```

Comment:
These last two need to be in the `preview` directory

Reply:
@jadrefke The CI validation noticed this too, along with some paths to example files that are apparently incorrect:

https://github.com/Azure/azure-rest-api-specs/pull/9857/checks?check_run_id=803989126

Please fix those too.